trinity/protocol/les/requests.py

```diff
@@ -1,20 +1,21 @@
 from typing import (
-    Any,
-    cast,
     Tuple,
 )
 
 from eth_typing import BlockIdentifier
 
 from eth.rlp.headers import BlockHeader
 
-from p2p.exceptions import ValidationError
-
-from trinity.protocol.common.requests import BaseHeaderRequest
+from trinity.protocol.common.requests import (
+    BaseHeaderRequest,
+)
 
 from .constants import MAX_HEADERS_FETCH
 
 
+BlockHeaders_R = Tuple[BlockHeader, ...]
```

Comment:
How is this used?

Reply:
Looks like another leftover from type hint hoop jumping.  Will remove.